### PR TITLE
Feature/malayalam language

### LINKED ISSUE #314: Malayalam language support (translation)
Adding malayalam language support (translation)

### PR DESCRIPTION
**Pull request by saranyaharidas**
Closes #314 
Adding malayalam language support (translation)

*Updated Overlap.js - Fix issue in button labels by taking from language

### CODE REVIEW

File: app/locales/ml/notification.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "notification_main_text": "നിങ്ങളുടെ ഇന്റർസെക്ഷൻ ദിവസം തിരിച്ച് പരിശോധിക്കുക",
+    "notification_title": "നിങ്ങളുടെ ഇന്റർസെക്ഷൻ വിശകലനം ചെയ്യുക",
+    "notification_data_not_available": "\n\nനിങ്ങൾക്ക് ട്രയൽ ഡാറ്റാ ഒന്നും ഡൌൺലോഡ് ചെയ്തിട്ടില്ല !\n\n",
+    "notification_warning_text": "ക്രമരഹിതമായ ഇന്റർസെക്ഷൻ സൃഷ്ടിക്കുന്നതിന് നിങ്ങൾക്ക് ഇനിപ്പറയുന്ന ബട്ടൺ അമർത്താം. മുന്നറിയിപ്പ്: ഇവ യഥാർത്ഥമല്ലാത്ത സിമുലേറ്റഡ് ഡാറ്റ ആണ് ",
```
Comment (alpita-masurkar) [inline app/locales/ml/notification.json:5]: You've excluded a newline character from this text:
"notification_warning_text": "You can press the following button to generate random intersections.\nWARNING: These are simulated intersections not real",

File: app/locales/ml/overlap.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "ഓവർലാപ്പ് പരിശോധിക്കുക",
+  "overlap_para_1": "പച്ച നടപ്പാത നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രത്തെ പ്രതിനിധീകരിക്കുന്നു. ഇളം പർപ്പിൾ സർക്കിളുകൾ പൊതു ഡാറ്റാഗണത്തെ പ്രതിനിധീകരിക്കുന്നു",
```
Comment (alpita-masurkar) [inline app/locales/ml/overlap.json:3]: and two more newline characters here...
 "overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset",
can you please add them?

File: app/locales/ml/notification.json (hunk 1 above)
Comment (saranyaharidas) [reply app/locales/ml/notification.json]: added ..

File: app/locales/ml/overlap.json (hunk 2 above)
Comment (saranyaharidas) [reply app/locales/ml/overlap.json]: done..

**Comment by saranyaharidas** [comment]
@alpita-masurkar @harshvitra   could you please check this

**Comment by kenpugsley** [comment]
Thank you for this translation.  The UI has changed significantly and has now finalized for the v1.0 release, which has resulted in a great deal of changes to the text in the final app.  Because of this we found that it was best to remove the existing translations and start again.  Even though we are also closing this PR, we hope you understand and will help us again.

The text in the app is now complete, so **we're asking for your help to redo this translation on the text for the v1.0 release**.  Our translation lead @alpita-masurkar will be coordinating this effort.

Thank you from the COVID Safe Paths dev team.

**Pull request closed without merging**